docs/content/latest/releases/_index.md

```diff
@@ -47,7 +47,7 @@ Support for YugabyteDB stable release series includes:
   * Yugabyte may direct Customers to Upgrade to a current release if a workaround does not exist.
```

Comment:
missing bracket in bullet under On this page

Reply:
I noticed that in some other PR and fixed it :)